examples/acq-environment.json

```diff
@@ -0,0 +1,10 @@
+{
+    "environment": {
+       "averageAirTemperature": 9.9,
+       "maxRelativeHumidity": 9.9,
+       "maxAltitude": 9.9,
+       "meteorologicalConditions": "meteorological conditions",
+       "solarAzimuth": 9.9,
+       "solarElevation": 9.9
+    }
+}
```

Comment:
The format of these examples should be to start with an array with a single object containing a single example of the class. For example, this one should look like this:

```
[{
       "averageAirTemperature": 9.9,
       "maxRelativeHumidity": 9.9,
       "maxAltitude": 9.9,
       "meteorologicalConditions": "meteorological conditions",
       "solarAzimuth": 9.9,
       "solarElevation": 9.9
}]